nestedtensor/nested/masking.py

```diff
@@ -14,15 +14,16 @@ def nested_tensor_from_padded_tensor(tensor, nested_dim=None, padding=-1):
     mask = (tensor != padding)
     return nested_tensor_from_tensor_mask(tensor, mask, nested_dim)
 
+"""
+Given a tensor t of size N_1 x N_2 x ... N_n and mask m of size M_1 x M_2 x ... M_d where d <= n
+return a NestedTensor of dimensionality d where subtensor t[i_1][i_2]...[i_d] is included as a Tensor constituent if m[i_1][i_2]...[i_d].
+The resulting NestedTensor will be of dimension n.
+Unless specific, it is attempted to yield a NestedTensor of minimal nested dimension.
+"""
 def nested_tensor_from_tensor_mask(tensor, mask, nested_dim=None):
-    """
-    Given a tensor t of size N_1 x N_2 x ... N_n and mask m of size M_1 x M_2 x ... M_d where d <= n
-    return a NestedTensor of dimensionality d where subtensor t[i_1][i_2]...[i_d] is included as a Tensor constituent if m[i_1][i_2]...[i_d].
-    The resulting NestedTensor will be of dimension n.
-    Unless specific, it is attempted to yield a NestedTensor of minimal nested dimension.
-    """
     # TODO: Should be possible with views only.
-    assert mask.dim() > 0
+    if not mask.dim() > 0:
+        raise RuntimeError("Mask has to have dimention > 0")
```

Comment:
I think technically this is supposed to be a ValueError too? Doesn't matter all that much.